starknet_py/net/account/account_client.py

```diff
@@ -469,6 +477,70 @@ async def execute(
     async def deploy(self, transaction: Deploy) -> DeployTransactionResponse:
         return await self.client.deploy(transaction=transaction)
 
+    async def deploy_contract(
+        self,
+        class_hash: Hash,
+        salt: Optional[int] = None,
+        unique: bool = True,
+        abi: Optional[list] = None,
+        constructor_calldata: Optional[Union[List[any], dict]] = None,
+        deployer_address: Optional[AddressRepresentation] = None,
+        max_fee: Optional[int] = None,
+        auto_estimate: bool = False,
+        version: Optional[int] = None,
```

Comment:
I'd consider combining some of these parameters into a TypedDict.

Reply:
#375